features/admin.tenants.v1/api/tenants.ts

```diff
@@ -160,3 +167,29 @@ export const getAssociatedTenants = (
             return Promise.reject(error?.response?.data);
         });
 };
+
+export const useDeploymentUnits =
```

Comment:
Let's move this to a dedicated file as that is the current convention for maintaining SWR hooks.

Reply:
Move to a new file